### ISSUE TITLE
Split 1.5.0 Swift SDK does not support Swift 4.2

### ISSUE THREAD

**Issue by bpollock-vida** [splitio/ios-client#63]
1. Install split via cocoapods
  pod 'Split', '~> 1.5.0'
2. Try to build app
3. Split SDK can not compile due to some UIApplication naming changes in Swift 4.2

**Comment by NicoZelaya** [comment]
Hey @bpollock-vida! 

We're sorry to hear you're experiencing issues, our team is taking a look and will get back to you as soon as we have an update. 

Thank you so much for your feedback!

Best,
Nico.

**Comment by SenhorCastor** [comment]
Hi bpollock-vida, to keep you up to date.

We are working on a version that will support version 4.2 of Swift and also backwards compatible with 4.0. We are aiming to have a release candidate version for tomorrow and run all our automation.

Best,
Fe

**Comment by SenhorCastor** [comment]
Hi bpollock-vida,

We just released version 1.5.1 that is compatible with swift 4.2 and 4.0.

Apologies for the delay, we ran into some updates on 4.2 (https://github.com/apple/swift-evolution/blob/master/proposals/0206-hashable-enhancements.md) that affected our hashing algorithms and we wanted to make sure that we kept consistency with previous releases and across sdks.

Let us know if this works, I will leave this open for some days

Thanks again for the feedback!
Fer
